ZKSyncSDK/Classes/Wallet/DefaultWallet+TransactionFee.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DefaultWallet+TransactionFee.swift
+//  ZKSyncSDK
+//
+//  Created by Eugene Belyakov on 16/01/2021.
+//
+
+import Foundation
+
+extension DefaultWallet {
+    
+    public func getTransactionFee(for transactionType:TransactionType,
+                                  tokenIdentifier: String,
+                                  completion: @escaping ZKSyncCompletion<TransactionFeeDetails>) {
+        self.getTransactionFee(for: transactionType,
```

Comment:
The address parameter is not the address of the sender, but the address of the recipient

Reply:
This is stub for the cases when there is no direct recipient of the transaction like Withdraw of ForcedExit. For transactions with recipient like Transfer there is another variant of getTransactionFee method that accepts recipient's address